lib/runtime/AutoPublicPathRuntimeModule.js

```diff
@@ -56,7 +56,7 @@ class AutoPublicPathRuntimeModule extends RuntimeModule {
 								"if(scripts.length) {",
 								Template.indent([
 									"var i = scripts.length - 1;",
-									"while (i > -1 && !scriptUrl) scriptUrl = scripts[i--].src;"
+									"while (i > -1 && (!scriptUrl || !scriptUrl.startsWith('http'))) scriptUrl = scripts[i--].src;"
```

Comment:
Sorry, we can't use `startsWith`, because we still support old browsers, we need to find another way

Reply:
Let's use regexp